projects.md

```diff
@@ -2,17 +2,26 @@
 title: Projects
 ---
 
-# Vac projects
+# Projects
 
-There are several projects under Vac.
+There are several projects under Vac Research.
 These are largely independent and can be seen as specific bets.
 
-## Waku Research
+## Secure Messaging
 
-Basically what Vac is primarily focused on today.
-Serves Waku Product, and aims to make Waku a self-sustained robust network with strong and modular privacy guarantees.
-This also includes secure messaging research and things like p2p/networking.
+We design modular, p2p protocols for private, anonymous, secure and censorship-resistant communications. The main focus is [Waku](https://waku.org), the communication layer for Web3.
 
-## Other projects
+## zk-WASM
 
-More projects will be added here soon.
+Research into making private and provable computation using WASM that is verified using zero-knowledge proofs.
+
+### RAD - RFCs & Dissemination
+
+**RFCs**: We build specifications for self-sustainable robust protocols with strong and modular privacy guarantees. Rough consensus and running code.
```

Comment:
Regarding "Rough consensus and running code."
I'd give credit to the [IETF Tao](https://www.ietf.org/about/participate/tao/).

Reply:
done